apps/schema/src/components/TableEditModal.vue

```diff
@@ -1,156 +1,164 @@
 <template>
-  <!-- when authorisation graphqlError show login-->
-  <Spinner v-if="loading" />
-  <div v-else-if="showLogin">
-    <MessageError v-if="graphqlError">{{ graphqlError }}</MessageError>
-    <SigninForm @login="loginSuccess" @cancel="cancel" />
-  </div>
-  <!-- when update succesfull show result before close -->
+  <IconAction
+    v-if="!show"
+    class="btn-sm hoverIcon"
+    icon="pencil-alt"
+    @click="show = true"
+  />
   <LayoutModal
-    v-else-if="success"
-    :title="title"
-    :show="true"
-    @close="$emit('close')"
+    v-else
+    :title="'Edit ' + table.name"
```

Comment:
This way the title changes when the users edits the name field. Shouldn't it be static until save is pressed?
Also when creating a table it says "Editing undefined" which ain't great.